### DISCUSSION TITLE
Width breakpoints not working with vite/react

### DISCUSSION THREAD

**Discussion by Calpurniax** [tailwindlabs/tailwindcss#13000] [Help] [answered]
hi! I created with Vite a React project and I'm styling a side-menu but I have problems with the breakpoints, I can see them in the inspector but it's crossed out. Other type of breakpoints are working just fine (I have a grid and a flex change) I have also another width change and works but it has no mobile value, just the "md:"
So this is my component
```
<nav className={ (isOpen? '-translate-x-0' : '-translate-x-full h-auto') +' text-white bg-black h-screen absolute top-0 ease-in-out duration-300 minScreen:w-full lg:w-60' }>  
        <ul className='flex flex-col justify-around uppercase w-full h-full items-center text-3xl relative top-0 '>
                <li className="hover:opacity-90"><Link to="/">Home</Link></li>
                <li className="hover:opacity-90"><Link to="/proyectos">Proyectos</Link></li>
                <li className="hover:opacity-90"><Link to="/contacto">Contacto</Link></li>
        </ul>
</nav>
```
  I added some extra config to the tailwind.config.js, so it looks like this:
```
  /** @type {import('tailwindcss').Config} */
export default {
  content: [
    "./index.html",
    "./src/**/*.{js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {
      height: {
        "screen/9": "90vh",
        "screen/8": "80vh",
        "screen/2": "50vh",
        "screen/3": "calc(100vh / 3)",
        "screen/4": "calc(100vh / 4)",
        "screen/5": "calc(100vh / 5)",
      },
      width: {
        "screen/2":"50vw",
        "screen/3":"calc(100vw / 3)",  
      },
      screens:{
        minScreen:{max:"1023px"}
      },
      keyframes:{
        slidein:{
          from:{
            opacity:"0",
            transform: "translateY(-10px)"
          },
          to:{
            opacity:"1",
            transform: "translateY(0)",
          },
        },
      },
      animation:{
        slidein300: "slidein 1s ease 300ms forwards",
        slidein900: "slidein 1s ease 900ms forwards"
      }
    },
  },
  variants: { display: ["responsive", "hover", "focus"] },
  plugins: [require("tw-elements-react/dist/plugin.cjs")],
}
```
The minScreen is the solution I found to make it works, so instead of `w-full lg:w-60` i have `minScreen:w-full lg:w-60`. That's what I mean with the crossed out styles
![imagen](https://github.com/tailwindlabs/tailwindcss/assets/60033000/af038020-e7d2-4e7b-bc7a-a46ed9cf309a)

**Comment by wongjn** [comment]
Consider providing a project that reproduces the unexpected behavior.

**Reply by Calpurniax** [reply]
Sorry Im newbie in these things, did u mean a repository?

**Reply by wongjn** [reply]
Yes,  a repository would be useful.

**Reply by Calpurniax** [reply]
I have created one with some code and extensions Im using in the project, and I have the same issue. This is the repo: https://github.com/Calpurniax/test_tailwind
thx!

**Reply by wongjn** [reply, marked as answer]
It is the CSS from `import "tw-elements-react/dist/css/tw-elements-react.min.css";` in `src/main.jsx` causing the issue. Consider removing this line and instead, add `"./node_modules/tw-elements-react/dist/js/**/*.js"` to the `content` file globs instead.

**Reply by Calpurniax** [reply]
ty! it works perfectly
